roles/configure-nexus/tests/ocp-sandbox-test.yml

```diff
@@ -0,0 +1,11 @@
+---
+# This test covers the full feature set provided by the role
+
+- name: "Configure Nexus"
+  hosts: nexus
+
+  vars:
+    nexus_url: http://nexus3-nexus.apps.s2.core.rht-labs.com
```

Comment:
could we remove the `rht-labs.com` portion here ? ... maybe just use `example.com` instead?